swift/README.md

```diff
@@ -1 +1,26 @@
-Curbmap have a Swift 4.x implementation cocoapod located [here](https://github.com/curbmap/OpenLocationCode-swift).
+# Swift Implementations of Open Location Code
+
+## Curbmap
+
+Curbmap have a Swift 4.x and 3.x implementation located [here](https://github.com/curbmap/OpenLocationCode-swift). 
```

Comment:
Nit: Curbmap has

Reply:
> It looks like the newly added implementation doesn't contain the changes mentioned in #190. Could you make those please?

Thanks for pointing this out. Working on it in https://github.com/google/open-location-code-swift/pull/4

Reply:
Done.